about.html

```diff
@@ -0,0 +1,188 @@
+        </li>
+      </ul>
+    </div>
+  </nav>
+  <main class="about-body container-fluid">
+    <section class="about-hero p-4">
+      <header class="about-title text-center mb-3">
+        <h1 class="the-title">Software Developers</h1>
+        <span>Global Summit 2020</span>
+      </header>
+      <div class="section-body about-text p-3">
+        <p>
+          This is an annual conference held at the end of every year, the top
+          leading software architects, dev teams & managers, gather to discuss
+          & discover the latest in developer technologies, languages,
+          platforms, and tools. Our conference & workshops invite you to get
+          intro lessons (or advanced tips and tricks) on the latest
+          technologies of the 21st century. Google is one of the best tech
+          companies that will be on the spotlight, presenting their tech &
+          business ideas to you.
+        </p>
+        <p>
+          In technology, there is incremental change & fundamental innovation.
+          Developer technology from artificial intelligence to big data
+          represents fundamental innovation that people can build on for
+          years. At this event, there will be companies like,
+          <a href="https://linkedin.com" target="_blank" rel="noopener noreferrer">LinkedIn</a>,
+          <a href="https://tedx.com" rel="noopener noreferrer">TedX</a> that will cover the full scope
+          of a software architect's job, from IT to leadership and business
+          skills.
+        </p>
+      </div>
+
+      <div class="questions text-center mt-4">
+        <p class="more">
+          Please contact us per Email for further questions & suggestions at
+          <br />
+          <span><a href="https://djpants90@yahoo.com" target="_blank" class="link-color" rel="noopener noreferrer">
+              DenzelNkomo@outlook.com <i class="fas fa-angle-right"></i></a></span>
+        </p>
+      </div>
+    </section>
+    <section class="past-events border-top py-4">
+      <header class="main-title d-flex justify-content-center align-items-center">
+        <h2 class="text-center text-muted">Past Events</h2>
+        <div class="underline text-center"></div>
+      </header>
+
+      <div class="past-events-wrapper row py-4 px-0 mx-0">
+        <p class="learn-more text-center col-12">
+          <a href="#" class="text-muted">Checkout these articles to learn more about our past conference
+            meetings. <i class="fas fa-angle-right"></i></a>
+        </p>
+        <div class="event-container row col-12 mt-3 px-3">
+          <div class="col-sm-12 col-lg-6">
+            <p class="events-text">
+              In 2014 we launched our first summit meeting with 10, 000 +
+              software developers, entreprenuers, managers & tech CEOs. TedX
+              discussed the best practices needed in mastering leadership
+              skills.
+            </p>
```

Comment:
- [ ] OPTIONAL: Your past events section did not follow the project requirement. In the original design it contains a section header, a divider, a line of text, and two images with text centered over them.

![image](https://user-images.githubusercontent.com/64894927/132728557-ab784de8-b4d5-4b4f-9bea-87a3da061591.png)

![image](https://user-images.githubusercontent.com/64894927/132728607-19d28d60-cd87-4a2f-b1c9-194307bace16.png)